app/assets/stylesheets/refills/_tables.scss

```diff
@@ -0,0 +1,95 @@
+.tables {
+  ///////////////////////////////////////////////////////////////////////////////////
```

Comment:
Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets